docs/user/migrating-to-v1.0.0.md

```diff
@@ -1,3 +1,37 @@
 # Migration to v1.0.0
 
-This release includes a breaking change **if you are upgrading from a version prior to `v1.0.0-beta.2`**. In the version `v1.0.0-beta.2` we introduced a change in MongoDB credentials generation that requires a migration hook that is only present from the version `v1.0.0-beta.2` to `v1.0.0-beta.4`. If you are upgrading Kubeapps from an older version please upgrade first to `v1.0.0-beta.4` to run the necessary migration job.
+This release includes several breaking changes that can make upgrading fairly
+difficult:
+
+- The AppRepository CRD is now installed using the `crd-install` Helm hook
+- The AppRepository objects are now installed as `pre-install` Helm hooks
+- Removal of a migration task to update the credential generation for MongoDB
+  that was introduced in the `v1.0.0-beta.4`
+
+If you have difficulty updating to the v1.0.0 release, we recommend backing up
+any AppRepository objects (custom repositories) you may have added and perform a
+clean install of Kubeapps.
+
+To backup a custom repository, run the following command for each repository:
+
+```
+kubectl get apprepsitory -o yaml <repo name> > <repo name>.yaml
```

Comment:
`s/apprepsitory/apprepository/` but I'd rather dump the whole list

Reply:
Dumping the whole list is not possible, due to `kubectl` complaining about the `resourceVersion` being newer when re-applying stable, bitnami, svc-cat and incubator (since the chart installs them). Well, it does technically work (you'd get the error on those repos, but any other custom ones would succeed), but it would look like something went wrong.